packages/doctype-caip10-link/src/caip10-link-doctype.ts

```diff
@@ -3,18 +3,27 @@ import {
   DoctypeConstructor,
   DoctypeStatic,
   DocOpts,
-  DocParams,
   Context,
-  CeramicCommit, UnsignedCommit,
+  CeramicCommit,
+  UnsignedCommit,
+  DocMetadata,
 } from '@ceramicnetwork/common';
 
 export const DOCTYPE_NAME = 'caip10-link'
 
 /**
  * Caip10Link parameters
  */
-export interface Caip10LinkParams extends DocParams {
+export interface Caip10LinkParams {
```

Comment:
Temporarily folding the fields from `DocParams` into this. My next PR will remove this type entirely as part of revamping the whole Caip10Link API